pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -264,15 +275,16 @@ def convert_nested_indexer(indexer_type, keys):
         tm.assert_series_equal(result, expected)
 
 
-@pytest.mark.parametrize('indexer, is_level1, expected_error', [
-    ([], False, None),  # empty ok
-    (['A'], False, None),
-    (['A', 'D'], False, None),
-    (['D'], False, r"\['D'\] not in index"),  # not any values found
-    (pd.IndexSlice[:, ['foo']], True, None),
-    (pd.IndexSlice[:, ['foo', 'bah']], True, None)
+@pytest.mark.parametrize('indexer, level, expected_error', [
```

Comment:
you can now drop the expected_error and remove the relevant code.

Reply:
indeed!